src/styles/_bootstrap_variables.scss

```diff
@@ -29,17 +30,17 @@ $red:    #CF4444 !default;
 $dark:   darken($blue, 17%) !default;
 
 $theme-colors: (
-        primary: $blue,
-        secondary: $gray-700,
-        success: $green,
-        info: $cyan,
-        warning: $yellow,
-        danger: $red,
-        light: $gray-100,
-        dark: $dark
+        "primary": $blue,
```

Comment:
why did you wrap it in the quotation marks? Are we sure it won't break other components which are using this `$theme-colors`?

Reply:
I just want to make it consistent